README.md

```diff
@@ -1,134 +1,193 @@
+- Implement Eiffel events vocabularies as described in [Eiffel-edition-paris](https://github.com/eiffel-community/eiffel/tree/edition-paris) 
 - Validate events' schema regarding target event’s version.
-- Provide  APIs for library users to publish, subscribe, and unsubscribe for strongly-typed Eiffel events to RabbitMQ.
+- Sign and verify events' signatures.
+- Serialization/deserialization of events.
+- Provide  APIs for users to publish, subscribe, acknowledge, reject and unsubscribe for strongly-typed Eiffel events to RabbitMQ.
 
-[TOC]
+It consists of two packages; 1) **Eiffel.Net** for events' implementation, 2) **Eiffel.RabbitMq.Client** for assisted publishing to RabbitMQ.
 
-## Services
 
-- **RabbitMQ**: service exists in `deploy/services/rabbitmq` directory.
 
-## Technologies
-
-- .NET 6
-- C# (9)
-- Docker  (19.03)
-- Docker Compose  (2.0.0)
-- Makefile
-
-### Dependencies
-
-- **RabbitMQ.Client 6.2.2**
-- **FluentResult 2.5.0**: An external library for  [Result](https://github.com/altmann/FluentResults) object implementation, NuGet [link](https://www.nuget.org/packages/FluentResults/).
-- **Newtonsoft.Json 13.0.1**
--  **[DocFX](https://dotnet.github.io/docfx/index.html)** documentation generation tool.
-
-
-
-## Requirements ##
-
-### For getting the application up and running ###
-
-- Makefile (<https://www.gnu.org/software/make/>).
-- Docker ([Installation](https://docs.docker.com/engine/installation/)).
-- Docker Compose ([Installation](https://docs.docker.com/compose/install/)).
-
-### For development ###
+# Requirements For Development #
 
 - .NET 6 ([Installation](https://dotnet.microsoft.com/download/dotnet/6.0)).
-- For installing needed .NET tools: `make dotnet-tool-restore`.
-
-## Quick Start ##
-
-1. Clone the repository.
-1. Switch to the development branch.
-
-## Configuration ##
-
-### RabbitMQ Configuration ###
+- C# (9)
 
-TBW
+## Dependencies
+
+- **RabbitMQ.Client 6.2.2**:  An external [library](https://github.com/rabbitmq/rabbitmq-dotnet-client) RabbitMQ communication, NuGet [link](https://www.nuget.org/packages/RabbitMQ.Client/).
+- **FluentResult 2.5.0**: An external [library](https://github.com/altmann/FluentResults) for Result object implementation, NuGet [link](https://www.nuget.org/packages/FluentResults/).
+- **Newtonsoft.Json 13.0.1**:  An external [library](https://github.com/JamesNK/Newtonsoft.Json) for serialization/deserialization support, NuGet [link](https://www.nuget.org/packages/Newtonsoft.Json).
+- **[DocFX](https://dotnet.github.io/docfx/index.html)** documentation generation tool.
+
+# Components
+
+## SDK Projects
+
+- [src/Eiffel.Net](src/Eiffel.Net): Events implementation.
+- [src/Eiffel.RabbitMq.Client](src/Src/Eiffel.RabbitMq.Client): Event Publishing service.
+
+## Tests
+
+A unit tests reside under [tests/Eiffel.Net.Tests](tests/Eiffel.Net.Tests) to test event validation, signing, verifying the signature, serialization, and deserialization.
+
+# Quick Start #
```

Comment:
I would like this chapter to be much closer to the top. I think this is the first thing many users want to see after reading the initial description.